apps/initializer/lib/initializer.ex

```diff
@@ -26,10 +26,16 @@ defmodule Initializer do
           Map.new(init_arg)
           |> Map.put(:supervisor_ref, supervisor_ref)
 
+        {:ok, state}
+      end
+
+      def handle_continue(:init, state) do
         case on_start(state) do
           {:ok, new_state} -> {:ok, new_state}
           {:error, reason} -> {:stop, reason}
         end
+
+        {:no_reply, state}
```

Comment:
get rid of this and just have the case statement return the `{:noreply, new_state}` or `{:stop, reason}`. The stop should cause things upstream to react since we're not expecting the Init servers to stop.